requirements.txt

```diff
@@ -1,29 +1,23 @@
-scipy~=1.7.0  # Not larger because of statsmodels
+# Base
+numpy>=1.21.0,<1.22  # Can't >=1.22 because of Numba
 pandas>=1.3.0
-seaborn
-matplotlib
-IPython
-joblib
-scikit-learn>=1.0.0
-ipywidgets
+scipy>=1.7.3
+ipython>=7.32.0
+joblib>=1.1.0
+scikit-learn>=1.0.2
+pyod>=0.9.8
+imbalanced-learn>=0.8.1
+category-encoders==2.4.0
+lightgbm>=3.0.0
+
+# Plotting
+matplotlib>=3.3.0
+scikit-plot>=0.3.7
 yellowbrick>=1.4
-lightgbm>=2.3.1
 plotly>=5.0.0
-pyyaml<6.0.0
-wordcloud
-textblob
-cufflinks>=0.17.0
-umap-learn
-pyLDAvis
-gensim<4.0.0
-spacy<2.4.0
-nltk
-mlxtend>=0.17.0
-pyod
-pandas-profiling>=2.8.0
-kmodes>=0.10.1
-mlflow
-imbalanced-learn>=0.8.1
-category-encoders>=2.2.2
-scikit-plot #for lift and gain charts
-Boruta
\ No newline at end of file
+
+# Time-series
+statsmodels>=0.13.2
```

Comment:
statsmodels~=0.12.1 because of issues with earlier versions

Reply:
what issues? All test pass on my pc with the newest version installed

Reply:
Must have been resolved now. I would keep it same as what sktime has it as.

https://github.com/alan-turing-institute/sktime/blob/f25e4d906e723700351fff664771cb28dd244bc9/pyproject.toml#L51